cfgov/unprocessed/apps/analytics-gtm/js/util/analytics-util.js

```diff
@@ -12,13 +28,61 @@ function addEventListenerToElem( elem, event, callback ) {
   }
 }
 
-const delay = ( function() {
+/**
+ * Log a message to the console if the `debug-gtm` URL parameter is set.
+ * @param {string} msg - Message to load to the console.
+ */
+function analyticsLog( ...msg ) {
+  if ( getQueryParameter( 'debug-gtm' ) === true ) {
+    console.log( `ANALYTICS DEBUG MODE: ${ msg }` );
+  }
+}
+
+/* Search for support of the matches() method by looking at
+   browser prefixes.
+   @param {HTMLNode} elem
+   The element to check for support of matches() method.
+   @returns {Function} The appropriate matches() method of elem. */
+function _getMatchesMethod( elem ) {
+  return elem.matches ||
+       elem.webkitMatchesSelector ||
+       elem.mozMatchesSelector ||
```

Comment:
I think we just need the ms prefix

Reply:
`elem.msMatchesSelector` on the line below?